### DISCUSSION TITLE
Login page as landing page

### DISCUSSION THREAD

**Discussion by ctayiz** [requarks/wiki#7321] [Help / Questions] [answered]
Hi Community, 
I have the problem that when I call up my wiki page, no login window is visible. I have set up local authentication.

I have searched for the setting but have not found it. Unfortunately, I can't find anything about this on the Internet.

I do not want to make the page immediately accessible from the outside. When the page is accessed, the login page should appear I do not want to make the page immediately accessible from the outside. When the page is accessed, the login page should appear.

Thank you.

**Comment by NGPixel** [comment, marked as answer]
Remove access from the Guests group.
